Web Server consolidation to just NGINX with uWSGI

```diff
@@ -0,0 +1,79 @@
+There has been some discussion recently on why the CKAN documentation and packages have two web servers (Apache and NGINX) installed as part of an initial CKAN installation. It seems this could be potentially confusing and we should really consolidate these two web server installs into just one.
+
+From the last Dev meeting (Thursday 3rd Oct) discussions had the following options:
+
+1. Drop NGINX
+2. Install NGINX plus uWSGI (ie: the uWSGI native protocol implemented in NGINX to serve Python apps)
+
+*** Option 2 was thought to be the best way to proceed ***
+
+This was some of the conversation on Thursday 3rd Oct
+
+	•	Nginx is really mostly used as a proxy as by default not much is cached by CKAN.
+	•	Maybe make it clearer that this is just a ‘basic suggest install’ to get customers going as ‘full production installations’ can vary considerably
+	•	Do we want to do some kind of survey to ask people how they've deployed so we can get a "typical" deployment?	Has this ever been done before?
+	•	Docker versions plus the Package version of the install will both need to be consistent with the new (updated) documentation
+
+Some thoughts to get the discussions happening:
+
+	•	We need to agree on the latest stable or mainline version or will this be a ‘Prebuilt Ubuntu Package’ from an Ubuntu Repository? This would make sense as it would have been thoroughly tested  
+	•	Do we have an HTTPS option OR do we make this default as everyone should really be using HTTPS these days. You can serve both HTTP and HTTPS requests from within the same virtual server so maybe this is the right idea. However then we would need to provide instructions to install an SSL certificate
+	•	Log file rotation for both the error log and the HTTP log - Perhaps we make sure this gets set up initially
+	•	Testing - I’m there will be a lot of testing to be done, What tests are required? Do we have some sort of testing software or framework already set up to test the Web servers?
+	•	Configure NGINX to automatically start during a reboot
+	•	What NGINX modules should be included by default?  SSL, obviously uWSGI
+
+We maybe could offer a bit more documentation on any advanced (less vanilla) installs like load balancing, session persistence OR maybe just a  link to the official NGINX documentation
```

Comment:
I think we should clearly document the minimal installation steps required on our supported Ubuntu versions but leave advanced instructions to the nginx docs or external blogs/wiki pages.

These days I think "minimal" includes letsencrypt. The last time I set that up there were quite a few extra steps to get nginx working (this was on an Amazon linux image, so it might be easier on Ubuntu)

Reply:
@wardi - yes this makes sense and will be simpler to document - It seems only you have taken a look at this pull request. Is there a better way to get the others involved in commenting on this document? or is everyone just too busy with other things? - thanks